### ISSUE TITLE
New version for the latest changes

### ISSUE THREAD

**Issue by mciordache** [percolate/fvd#4]
Hi guys,

Can you please create a new release/tag for the latest changes. 
I have a error: "Unsupported engine for fvd@1.0.0: wanted: {"node":"0.10.x"}" 
I think you removed the engine restriction in the latest PR ?

**Comment by boutell** [comment]
Yes, is this something you might be able to do? 🙏 The package passes its tests fine in node 18 and node 20, the currently supported major versions. All it needs is an `npm publish` to clear the scary warning 😄 

Thanks for all you do!
